### ISSUE TITLE
Invite email link sometimes fails

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-app#351]
Nancy R Lange was invited to join CMS, but when she clicked the accept link from MS Outlook on a Mac she had this error.  I can reproduce this by clicking an "accept invite" link a second time (already been used). We need to handle the case when a user clicks on a used or invalid "accept invite" link.

But strangely, this person didn't already click the link.She is still in the "Invited Users" section (see image at bottom).  It was the first time the link was clicked.  What can we say to Maude to invite this user?

![Image](https://github.com/culturecreates/footlight-app/assets/419491/bceea55b-8a7d-4974-bc4d-1686e75d33a8)

![Image](https://github.com/culturecreates/footlight-app/assets/419491/3581967a-ff14-4973-86d0-a6cbde71da26)
![Image](https://github.com/culturecreates/footlight-app/assets/419491/df88befe-4cc8-421b-a27e-3a48bb88a19e)

**Comment by dev-aravind** [comment]
I could not reproduce this issue with this specific invitation: 646cde0c93ec11006414ff43.
See the screenshot of how it looks to me when I give the URL: https://cms.footlight.io/join?invitationId=646cde0c93ec11006414ff43
<img width="1431" alt="invitation-details-from-databse" src="https://github.com/culturecreates/footlight-app/assets/132546883/7ce9103e-f023-4926-8f2a-b0741ebeda9f">
<img width="1512" alt="invitation-screenhsot-web" src="https://github.com/culturecreates/footlight-app/assets/132546883/ab28a198-9dd4-47c1-8a37-074768231c0c">

**Comment by dev-aravind** [comment]
I also looked into the code and verified that, if a user tries to accept the invitation "already accepted", then the backend sends the response: 
`"The invite has been already accepted by the user. Please contact ${contactEmail}". `

**Comment by saumier** [comment]
@AbhishekPAnil  When there is a backend message when should display it after the friendly message. So in this case, the error should be: 

```
**Something went wrong.** 
Error: "The invite has been already accepted by the user. Please contact ${contactEmail}".
```

**Comment by saumier** [comment]
@troughc For situations where emails are lost (spam or other) then there should be a way for the admin to help out the user. One way to do this is to add a button "Accept invitation" into the CMS. This button would prompt for a password and then add the user to the calendar. The admin could then ask them to reset their password later.

**Comment by saumier** [comment]
@SuhailAliyar I noticed that if an admin cancels an invite, the user's email remains active in the system but there is no where to see them. Shouldn't the user's email be deleted from the system if they are the last user (no other calendars)? The only other option is a section called "uninvited".  I also noticed that if a new invite is made to the same email address but to a different first name and last name,  then the original first name and last name reappears in the CMS invited list. Maybe deleting the user after uninviting them (if they are the last user) would fix this.
